lib/replicator.js

```diff
@@ -1161,6 +1171,8 @@ class Peer {
   }
 
   async _send (req) {
+    this.tracer.trace()
```

Comment:
move this one here https://github.com/holepunchto/hypercore/pull/465/files#diff-3344a84994d8e172768f33ec4bf514ba2bc28eb7eeffff10fcbbd294efe56edcR1204 and pass the req as props